### ISSUE TITLE
Can not set url via {:system, "REDIS_URL"}

### ISSUE THREAD

**Issue by kaelumania** [akira/exq#267]
I can not set the Redis url using the runtime config injection via `{:system, "REDIS_URL"}`

```elixir
config :exq,
  name: Exq,
  namespace: "exq",
  concurrency: 2, # lets no overwhelm rails
  queues: ["default"],
  url: {:system, "REDIS_URL"},
  poll_timeout: 50,
  scheduler_poll_timeout: 200,
  scheduler_enable: true,
  max_retries: 25,
  shutdown_timeout: 5000
```

I get the following error:

```
** (Mix) Could not start application exq: Exq.start(:normal, []) returned an error: shutdown: failed to start child: Redix
    ** (EXIT) an exception was raised:
        ** (FunctionClauseError) no function clause matching in Redix.Utils.sanitize_starting_opts/2
            (redix) lib/redix/utils.ex:35: Redix.Utils.sanitize_starting_opts({:system, "REDIS_URL"}, [backoff: 100, timeout: 5000, name: Exq.Redis.Client])
            (redix) lib/redix/connection.ex:35: Redix.Connection.start_link/2
            (stdlib) supervisor.erl:365: :supervisor.do_start_child/2
            (stdlib) supervisor.erl:348: :supervisor.start_children/3
            (stdlib) supervisor.erl:314: :supervisor.init_children/2
            (stdlib) gen_server.erl:328: :gen_server.init_it/6
            (stdlib) proc_lib.erl:247: :proc_lib.init_p_do_apply/3
```

It seems that Redix does not support this way of configuration, therefore `Exq` needs to sanitize it upfront.

**Comment by kaelumania** [comment]
My apologies, seems that this behaviour was introduced just a few weeks ago and since then there hasn't been an `Exq` release.

**Comment by kaelumania** [comment]
Further here is a PR to support default values for this pattern
https://github.com/akira/exq/pull/268

**Comment by akira** [comment]
@kaelumania just published 0.8.7 along with your merged PR.  Thanks!
